domains/ignasius.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ace3",
+        "email": "ace3xx@gmail.com"
+    },
+    "record": {
+        "A": ["138.2.84.251"]
```

Comment:
I cannot reach your website
"404 Not Found"

Reply:
Hi, sorry, i forget to reload the nginx service.
right now, i already reload the config.


thanks